src/types/scriptProviderTypes.ts

```diff
@@ -45,3 +46,5 @@ export interface ScriptProviderProps {
     children?: React.ReactNode;
     deferLoading?: boolean;
 }
+
+export type PayPalProviderChildren = ReactElement | ReactPortal | ReactFragment;
```

Comment:
I'd prefer to keep the name general.
```suggestion
export type Children = ReactElement | ReactPortal | ReactFragment;
```